File: Jour15/jour15.c
#include<stdio.h>

int main()
{
	int i, n, som =0;
	printf("Ce programme affiche la somme des n premiers entiers naturels.\n");
	do
	{
		printf("Donner n : ");
		scanf("%d",&n);
		if(n < 0|| n != (int)n)
			printf("Erreur n est un entier naturel.\n");
	}while(n < 0 || n != (int)n);

	for(i = 1; i <= n; i++)
		som += i;

	printf("La somme des %d premier(s) termes vaut : %d\n",n,som);
	return	0;
}

File: Jour15/jour15.py
#Coding : utf-8
print("Ce programme affiche la somme des n premier entier naturels.")
while 1:
	n = input("Donner n : ")
	try:
		n = int(n)
		if(n < 0):
			print("Erreur :::: n est un entier naturel.")
			continue
	except:
		print("Saisie invalide!!!")
		continue
	else:
		break

som = 0
for i in range(1,n+1):
	som += i
print(f"La somme des {n} premier temes vaut : {som}")